draft-ietf-ppm-dap.md

```diff
@@ -1782,25 +1804,27 @@ Next, the Helper checks that:
 
 If any of these checks fail, then the Helper MUST abort with error
 `invalidMessage`. Additionally, if any prep step appears out of order relative
-to the previous request, then the Helper MAY abort with error `invalidMessage`.
-(Note that a report may be missing, in which case the Helper should assume the
-Leader rejected it.)
-
-[OPEN ISSUE: Issue 438: It may be useful for the Leader to explicitly signal
-rejection.]
+to the previous request, then the Helper MUST abort with error
+`invalidMessage`. (Note that a report may be missing, in which case the Helper
+should assume the Leader rejected it.)
 
 Next, the Helper checks if the continuation step indicated by the request is
 correct. (For the first `AggregationJobContinueReq` the value should be `1`;
 for the second the value should be `2`; and so on.) If the Leader is one step
 behind (e.g., the Leader has resent the previous HTTP request), then the Helper
-MAY attempt to recover by re-sending the previous `AggregationJobResp`. In this
-case it SHOULD verify that the contents of the `AggregationJobContinueReq` are
-identical to the previous message (see {{aggregation-step-skew-recovery}}).
-Otherwise, if the step is incorrect, the Helper MUST abort with error
-`stepMismatch`.
+MAY attempt to recover by sending HTTP 202 status Accepted and discontinuing
+any further work. In this case it SHOULD verify that the contents of the
+`AggregationJobContinueReq` are identical to the previous message (see
+{{aggregation-step-skew-recovery}}). Otherwise, if the step is incorrect, the
+Helper MUST abort with error `stepMismatch`.
+
+If the Helper finds the `AggregationJobContinueReq` to be valid, it immediately
+responds with HTTP status 202 Accepted. The Helper is now ready to continue
+preparation for each report, asynchronously from any further requests from the
+Leader.
```

Comment:
Same nit as above - this wording frames it as if the preparation the reports are asynchronous from each other, when it's really the jobs as the base unit of asynchronicity